Watchman.Engine/Alarms/Defaults.cs

```diff
@@ -712,7 +717,8 @@ public static class Defaults
                 DimensionNames = new[] { "LoadBalancerName" },
                 ComparisonOperator = ComparisonOperator.GreaterThanOrEqualToThreshold,
                 Statistic = Statistic.Average,
-                Namespace = AwsNamespace.Elb
+                Namespace = AwsNamespace.Elb,
+                TreatMissingData = TreatMissingDataConstants.Breaching
```

Comment:
Does this statistic always exist and we are setting it to `TreatMissingDataConstants.Breaching`?

Reply:
I think this is just wrong tbh, will fix